pysph/examples/gas_dynamics/accuracy_test_2d.py

```diff
@@ -0,0 +1,165 @@
+"""
```

Comment:
Each example's docstring should follow a simple convention, see this for example:
https://github.com/pypr/pysph/blob/master/pysph/examples/dam_break_2d.py

The first line basically should be a short description with an indication of the time required to run the example on a single core.